conf/broker.conf

```diff
@@ -37,8 +37,11 @@ webServicePortTls=8443
 # Enable the WebSocket API service in broker
 webSocketServiceEnabled=false
 
-# Control whether to bind directly on localhost rather than on normal hostname
-bindOnLocalhost=false
+# Hostname or IP address the service binds on, default is InetAddress.getLocalHost().getHostName().
+bindAddress=
+
+# Hostname or IP address the service advertises to the outside world. If not set, the value of bindAddress is used.
+advertisedAddress=
```

Comment:
Addressed in https://github.com/yahoo/pulsar/pull/26/commits/37450339cc00d11793ebef16b9a3959526bc8825